src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/softmax/softmax_kernel_blocked.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "softmax_kernel_blocked.h"
+#include "kernel_selector_utils.h"
+
+namespace kernel_selector {
+ParamsKey SoftmaxKernelBlocked::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::INT8);
+    k.EnableOutputDataType(Datatype::UINT8);
```

Comment:
Softmax only supports fp by nature. Could you get rid of all the integer supports?

Reply:
Removed INT.